history-microservice/Dockerfile

```diff
@@ -33,6 +33,18 @@ LABEL org.opencontainers.image.description="A microservice prepared for monitori
 ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
+RUN git clone https://github.com/idrinth-api-bench/history-microservice.git \
```

Comment:
cloning into somewhere, please supply a target

Reply:
added target for clone